frontend/src/pages/article/components/article-form/article-form.tsx

```diff
@@ -39,12 +35,11 @@ type Properties = {
 const ArticleForm: React.FC<Properties> = ({ articleForUpdate }) => {
   const navigate = useNavigate();
   const dispatch = useAppDispatch();
-  const { generatedPrompt, saveArticleStatus } = useAppSelector(
-    ({ prompts, articles }) => ({
-      generatedPrompt: prompts.generatedPrompt,
-      saveArticleStatus: articles.saveArticleStatus,
-    }),
-  );
+  const [isPublishing, setIsPublishing] = useState(false);
+  const [isSavingDraft, setIsSavingDraft] = useState(false);
```

Comment:
Can't we store only what type of request was fired and use it in combination with `saveArticleStatus` for `isLoading`?